nix/bazel.nix

```diff
@@ -24,12 +24,23 @@ let shared = rec {
     openssl
     gnupatch
     patchelf
-    postgresql_9_6
     protobuf3_8
     python3
     toxiproxy
     zip
-    ;
+  ;
+
+    postgresql_9_6 = if pkgs.buildPlatform.libc == "glibc"
+      then pkgs.runCommand "postgresql_9_6_wrapper" { buildInputs = [ pkgs.makeWrapper ]; } ''
+      mkdir -p $out/bin $out/include $out/lib $out/share
```

Comment:
I guess the symlink overwrites it? good catch

Reply:
I tried this out, what happens is that `ln -s` places the symlink into the previously created directory, instead of replacing it with the symlink. I.e. this
```
$ mkdir foo bar; ln -s foo bar; tree
.
├── bar
│   └── foo -> foo
└── foo
```
instead of this
```
$ mkdir foo; ln -s foo bar; tree
.
├── bar -> foo
└── foo
```

Reply:
I’m very confused, how did this even work then … The tests passed

Reply:
I'm guessing the latter three dirs are effectively unused. Some grepping suggests that we only depend on the binaries provided by that Nix derivation directly in the daml codebase. These binaries might depend on shared libraries, but their `RPATH` will point to the original Nix store path, not the new wrapper store path.

Reply:
yeah could be. confusingly postgres itself complains if they don’t exist but apparently it doesn’t check more than this.